File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/JsonPrimitive.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class JsonPrimitive 
{
    public int AsInt { get; set; }
    public bool Number { get; set; }
    public bool Boolean { get; set; }
    public string AsString { get; set; }
    public bool AsBoolean { get; set; }
    public Object AsNumber { get; set; }
    public double AsDouble { get; set; }
    public float AsFloat { get; set; }
    public long AsLong { get; set; }
    public byte[] AsByte { get; set; }
    public string AsCharacter { get; set; }
    public decimal AsBigDecimal { get; set; }
    public int AsBigInteger { get; set; }
    public int AsShort { get; set; }
    public bool String { get; set; }
    public JsonObject AsJsonObject { get; set; }
    public bool JsonNull { get; set; }
    public bool JsonArray { get; set; }
    public bool JsonObject { get; set; }
    public bool VarJsonPrimitive { get; set; }
    public JsonArray AsJsonArray { get; set; }
    public JsonPrimitive AsJsonPrimitive { get; set; }
    public JsonNull AsJsonNull { get; set; }
}

File: server-scala-play-server/app/model/JsonPrimitive.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for JsonPrimitive.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class JsonPrimitive(
  asInt: Option[Int],
  number: Option[Boolean],
  boolean: Option[Boolean],
  asString: Option[String],
  asBoolean: Option[Boolean],
  asNumber: Option[JsObject],
  asDouble: Option[Double],
  asFloat: Option[Float],
  asLong: Option[Long],
  asByte: Option[ArrayByte],
  asCharacter: Option[String],
  asBigDecimal: Option[BigDecimal],
  asBigInteger: Option[Int],
  asShort: Option[Int],
  string: Option[Boolean],
  asJsonObject: Option[JsonObject],
  jsonNull: Option[Boolean],
  jsonArray: Option[Boolean],
  jsonObject: Option[Boolean],
  jsonPrimitive: Option[Boolean],
  asJsonArray: Option[JsonArray],
  asJsonPrimitive: Option[JsonPrimitive],
  asJsonNull: Option[JsonNull]
)

object JsonPrimitive {
  implicit lazy val jsonPrimitiveJsonFormat: Format[JsonPrimitive] = Json.format[JsonPrimitive]
}

